### DISCUSSION TITLE
IP Access List Issue

### DISCUSSION THREAD

**Discussion by mrozj** [NginxProxyManager/nginx-proxy-manager#1509] [Q&A]
NGINX newbie here. I setup my first NGINX install a few weeks back. Everything works great except for setting IP based access lists. I enter my class B subnet 10.10.0.0/16. Both NGINX and my client machine are on that subnet. I only receive 403 Forbidden errors. If I enter a username/password rule that works fine but I cannot get IP based access lists to work. Any suggestions?

**Comment by the1ts** [comment]
Not sure if this is the case, but if you are using nat hairpinning or nat loopback on your router, you will appear to nginx as coming from the inside IP of the router not the IP of your client machine, this may not be in the same subnet.
